src/lib.rs

```diff
@@ -1,3 +1,5 @@
+extern crate core;
+
```

Comment:
Is this deliberate? I don't see `core::..` used anywhere, though googling shows people sometimes need to use it for type conversions?

Reply:
Nope, either CLion or I added it accidentally.